.github/workflows/linux_meson.yml

```diff
@@ -16,6 +16,7 @@ defaults:
 
 env:
   PYTHON_VERSION: 3.11
+  NPY_USE_BLAS_ILP64: 1
```

Comment:
This does nothing right now, so best not to add it here. Let's get the 32-bit build to work first; ILP64 is still WIP on all platforms.

Reply:
ok